docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -342,24 +353,60 @@ the current running process:
     Size in bytes of virtual memory that is guaranteed to be available to the 
     running process.
 
+
+[[cluster-nodes-stats-api-response-body-ingest]]
+===== `ingest` section
+
 The `ingest` flag can be set to retrieve statistics that concern ingest:
 
 `ingest.total.count`::
-    The total number of document ingested during the lifetime of this node
+    (integer)
+    Total number of documents ingested during the lifetime of this node.
 
 `ingest.total.time_in_millis`::
-    The total time spent on ingest preprocessing documents during the lifetime 
-    of this node.
+    (integer)
+    Total time spent preprocessing ingest documents during the lifetime of this
+    node.
 
 `ingest.total.current`::
-    The total number of documents currently being ingested.
+    (integer)
+    Total number of documents currently being ingested.
 
 `ingest.total.failed`::
-    The total number ingest preprocessing operations failed during the lifetime 
-    of this node.
+    (integer)
+    Total number of failed ingest operations during the lifetime of this node.
+
+`ingest.pipelines.<pipeline-id>.count`::
+    (integer)
+    Number of documents preprocessed by the ingest pipeline.
 
-On top of these overall ingest statistics, these statistics are also provided on 
```

Comment:
I didn't feel this sentence was needed anymore, but let me know if the `<pipeline-id>` convention is intuitive.
I can re-add if needed.